lib/core/utils/get-scroll.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Get the scroll position of given element
+ * @method getScroll
+ * @memberof axe.utils
+ * @param {Element} node
+ * @param {buffer} (Optional) allowed negligence in overflow
+ * @returns {Object | undefined}
+ */
+axe.utils.getScroll = function getScroll(elm, buffer = 0) {
+	if (
+		elm.scrollHeight < elm.clientHeight ||
```

Comment:
This should be `elm.scrollHeight <= elm.clientHeight + buffer`.
1. Use `<=` instead of `<`. scroll is *never* smaller than client
2. If you add your buffer here, you don't need a second if statement. Also much less confusing about what that buffer is supposed to do.